server/app/services/telegram_bot_service.py

```
@@ -0,0 +1,442 @@
+            "🇺🇸 **Luke**      - Cheerful US follow-up agent\n"
+            "🇺🇸 **Jessica**   - Enthusiastic US feedback agent\n\n"
+            "👆 **Click on any agent above to start your demo call!**"
+        )
+        
+        await update.message.reply_text(
+            agent_selection_message,
+            parse_mode='Markdown',
+            reply_markup=reply_markup
+        )
+    
+    async def button_callback(self, update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
```

Ruff (0.14.7)

Unused method argument: `context`

(ARG002)